docs/source/developer/distributions.rst

```diff
@@ -0,0 +1,232 @@
+- :meth:`~metasyn.distribution.BaseDistribution.matches_name`: Class method to check if a distribution matches a given name (specified in the ``implements`` field).
+- :meth:`~metasyn.distribution.BaseDistribution.default_distribution`: Abstract class method
+- ``__str__``: Overridden method to return a formatted string representation of the distribution.
+
+.. warning:: 
+  When implementing a new distribution, the :meth:`~metasyn.distribution.BaseDistribution._fit`, :meth:`~metasyn.distribution.BaseDistribution.draw`, and :meth:`~metasyn.distribution.BaseDistribution.to_dict` methods *must* be implemented. 
+
+ScipyDistribution class
+^^^^^^^^^^^^^^^^^^^^^^^
+The :class:`~metasyn.distribution.ScipyDistribution` is a specialized base class for distributions that are based on
+`SciPy <https://docs.scipy.org/doc/scipy/index.html>`_ statistical distributions. 
+
+All the :mod:`~metasyn.distribution.datetime`, :mod:`~metasyn.distribution.discrete`, and :mod:`~metasyn.distribution.continuous` distributions are derived from this class.
+
+
+UniqueDistributionMixin class
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The :class:`~metasyn.distribution.UniqueDistributionMixin` is a mixin class that can be combined with other distribution classes to create distributions that generate unique values.
+
+For example, the unique variants of the :class:`metasyn.distribution.regex.RegexDistribution` and the :class:`metasyn.distribution.faker.UniqueFakerDistribution` are implemented using this mixin as follows:
+
+.. code-block:: python
+
+    @metadist(implements="core.unique_regex", var_type="string", is_unique=True)
+    class UniqueRegexDistribution(UniqueDistributionMixin, RegexDistribution):
+
+.. code-block:: python
+
+    @metadist(implements="core.unique_faker", var_type="string")
+    class UniqueFakerDistribution(UniqueDistributionMixin, FakerDistribution):
+
+
+Metadist decorator method
+^^^^^^^^^^^^^^^^^^^^^^^^^
+When implementing a new distribution, the ``metadist`` decorator helps set the attributes of that distribution (e.g. ``implements``, ``var_type``, etc.). 
+
+**Parameters:**
+
+- ``implements``: A unique name for the distribution type, e.g. ``core.uniform`` or ``core.poisson``. 
+- ``var_type``: The type of variable associated with the distribution, e.g. ``discrete`` or ``continuous``.
+- ``provenance``: Information about the source (core, plugin, etc.) of the distribution.
+- ``privacy``: The privacy class or implementation associated with the distribution.
+- ``is_unique``: A boolean indicating whether the values in the distribution are unique.
+- ``version``: The version of the distribution. 
+
+.. admonition:: Note
+
+    Note that the parameters are the same as the attributes of the :class:`~metasyn.distribution.BaseDistribution` class.
```

Comment:
I think this might some clarification. Maybe that it simply sets the class attributes of the BaseDistribution? Now it sounds a bit like it is an accident or something.

Reply:
You're right. I have opted to remove the parameter bit altogheter and rephrase the rest, as it was a bit redundant anyways.